tests/integration/suite/daprd/metrics/httpserver_lowcardinality.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	procdaprd "github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(httpServerLowCardinality))
+}
+
+// httpServerLowCardinality tests daprd metrics for the HTTP server configured with low cardinality
+type httpServerLowCardinality struct {
+	base
+}
+
+func (m *httpServerLowCardinality) Setup(t *testing.T) []framework.Option {
+	configFile := filepath.Join(t.TempDir(), "config.yaml")
+	require.NoError(t, os.WriteFile(configFile, []byte(`
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: lowcardinality
+spec:
+  metrics:
+    http:
+      increasedCardinality: false
```

Comment:
Please also have a test for when this value is nil.

Reply:
Added test to `pkg/config/configuration_test.go` since the value will change in the future and the test is duplicate.